magicctapipe/scripts/mars/tests/test_compare_params.py

```diff
@@ -0,0 +1,223 @@
+            test_calibrated_mars_M1_simulated[i],
+            test_calibrated_mars_M2_simulated[i],
+            test_superstar_mars_simulated[i],
+        )
+    )
+
+for i in range(len(test_calibrated_mars_M1_real)):
+    file_list.append(
+        (
+            test_calibrated_mars_M1_real[i],
+            test_calibrated_mars_M2_real[i],
+            test_superstar_mars_real[i],
+        )
+    )
+
+
+@pytest.mark.parametrize(
+    "dataset_calibrated_M1, dataset_calibrated_M2, dataset_superstar",
+    file_list,
+)
+def test_compare_hillas_stereo_params(
+    dataset_calibrated_M1, dataset_calibrated_M2, dataset_superstar, tmp_path
+):
+    from magicctapipe.scripts.lst1_magic import (
+        magic_calib_to_dl1,
+        merge_hdf_files,
+        stereo_reconstruction,
+    )
+
+    from ctapipe_io_magic import MAGICEventSource
+
+    params_list = [
+        "width_M1",
+        "width_M2",
+        "length_M1",
+        "length_M2",
+        "size_M1",
+        "size_M2",
+        "delta_M1",
+        "delta_M2",
+        "cogx_M1",
+        "cogx_M2",
+        "cogy_M1",
+        "cogy_M2",
+        "corex",
+        "corey",
+        "az",
+        "zd",
+        "hmax",
+        "impact_M1",
+        "impact_M2",
+        "slope_M1",
+        "slope_M2",
+    ]
+
+    source = MAGICEventSource(
+        input_url=dataset_calibrated_M1,
+        process_run=False,
+    )
+
+    is_mc = source.is_simulation
+    run_number = source.run_numbers[0]
+
+    if is_mc:
+        config_params = {
+            "magic-cta-pipe-input": {
+                "MCP-path": str(
+                    test_data / "simulated/dl1_stereo" / f"dl1_stereo_magic_only_MAGIC_GA_za35to50.Run{run_number}.h5"
+                )
+            },
+            "MARS-input": {"MARS-path": str(dataset_superstar)},
+            "Params": params_list,
+            "Output_paths": {
+                "file_output_directory": str(test_data / "simulated/test_params"),
+                "image_output_directory": str(test_data / "simulated/test_params"),
+            },
+        }
+    else:
+        config_params = {
+            "magic-cta-pipe-input": {
+                "MCP-path": str(
+                    test_data / "real/dl1_stereo" / f"dl1_stereo_magic_only_MAGIC.Run0{run_number}.h5"
+                )
+            },
+            "MARS-input": {"MARS-path": str(dataset_superstar)},
+            "Params": params_list,
+            "Output_paths": {
+                "file_output_directory": str(test_data / "real/test_params"),
+                "image_output_directory": str(test_data / "real/test_params"),
+            },
+        }
+
+    config_params_file = str(tmp_path / "compare_params_config.yaml")
+
+    with open(config_params_file, "w") as outfile:
+        yaml.dump(config_params, outfile, default_flow_style=False)
+
+    config_mcp = {
+        "mc_tel_ids": {"LST-1": 1, "MAGIC-I": 2, "MAGIC-II": 3},
+        "MAGIC": {
+            "image_extractor": {
```

Comment:
is the image_extractor and charge_correction needed here? the MARS files are already calibrated and in MCP this part is only needed for sim_telarray files processing